nototools/generate_website_2_data.py

```diff
@@ -554,7 +554,7 @@ def get_css_generic_family(family):
 
 CSS_WEIGHT_MAPPING = {
     'Thin': 250,
```

Comment:
Ok. 
I know of DemiLight (350). It's added to match the lighter of Roboto and Noto Sans (I forgot which) while 400 match the thicker of the two. 
Anyway, did you change ExtraLight to 250 as well?  With that, LGTM